actian_odbc/dialect.tdd

```diff
@@ -104,6 +104,12 @@
 		<formula>NVL2(%1,0 , 4) = 4</formula> 
 		<argument type='date' />
     </function>       
+
+   <function group='logical' name='ISNULL' return-type='bool'>
```

Comment:
good catch! I have to confess the last time I did a diff, I think I only diff'd the XML files not the tdd files. There are other differences (for example ROUND), I believe these files should be the same for both JDBC and ODBC.

Looks like we got lucky with the test suite detecting this.

Reply:
Any luck with this?

Reply:
Commit https://github.com/ActianCorp/actian_tableau_connector/pull/54/commits/c9da17b06d7fc281009c9098608cf435a37a5fb0 syncs the JDBC and ODBC dialect.tdd files. The overall TDVT test results for both connectors are unaffected.

    diff actian_jdbc\dialect.tdd actian_odbc\dialect.tdd
    3c3
    <          class='actian_jdbc'
    ---
    >          class='actian_odbc'

Connector | Passed | Failed | Total Tests
--|--|--|--
JDBC | 844 | 29 | 873
ODBC | 832 | 41 | 873